networking/network_observability/installing-operators.adoc

```diff
@@ -5,7 +5,7 @@ include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
-Installing Loki is a prerequisite for using the Network Observability Operator. It is recommended to install Loki using the Loki Operator; therefore, these steps are documented below prior to the Network Observability Operator installation.
+Installing Loki is a recommended prerequisite for using the Network Observability Operator. Therefore, these steps are documented below prior to the Network Observability Operator installation. You can choose to use xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this. 
```

Comment:
Installing Loki in general or the Loki Operator or is the operator implied?

Reply:
More direct:

> Before you install the Network Observability Operator, consider completing the steps outlined in the following procedure.

^^ Move sentence to end of intro.

Reply:
@skrthomas  Installing the Loki Operator installs Loki, right?